frontend/src/components/ShootAccessRestrictions/AccessRestrictionChips.vue

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: Apache-2.0
 
 <template>
   <div>
-    <v-row v-for="{ key, title, description, options } in selectedAccessRestrictions" :key="key" class="mx-0 mb-1">
+    <div v-for="{ key, title, description, options } in selectedAccessRestrictions" :key="key" class="d-flex">
```

Comment:
Why do we need the outer div? Isn't one div sufficient?

Reply:
The outer div is because the inner div is in a v-for loop and template must contain only one element. However, the v-row caused layout issues when used inside the v-list component. So I replaced it with div

Reply:
The outer one adds the row layout